cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/ProgramScheduleStoreDataset.java

```diff
@@ -112,28 +111,32 @@
    * @return the lexicographically largest namespace id String with schedule migration completed
    */
   public String migrateFromAppMetadataStore(NamespaceId namespaceId, Store appMetaStore) {
-    String completedNamespace = getMigrationCompleteNamespace();
-    if (completedNamespace != null && completedNamespace.compareTo(namespaceId.toString()) > 0) {
-      return completedNamespace;
+    try {
+      String completedNamespace = getMigrationCompleteNamespace();
+      if (completedNamespace != null && completedNamespace.compareTo(namespaceId.toString()) > 0) {
+        return completedNamespace;
+      }
+    } catch (Exception e) {
```

Comment:
so just let it throw exception? actually I forgot to remove `return` in the next line, and will let migration continue in case any exception happens here